activesupport/lib/active_support/core_ext/enumerable.rb

```diff
@@ -97,7 +97,7 @@ def many?
     end
   end
 
-  # Returns a new array that includes the passed elements.
+  # Returns a new enumerable that includes the passed elements.
```

Comment:
[2] pry(main)> [ 1, 2, 3 ].including(4, 5).class
=> Array

Reply:
My bad. Thank you for caching that.